### PR TITLE
Added a via keymap folder for n87

### PR DESCRIPTION
**Pull request by NightlyBoards**
## Description

Added a via keymap folder. It's tested working with the via software.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/nightly_boards/n87/keymaps/via/rules.mk [hunk 1]
```diff
@@ -0,0 +1,3 @@
+VIA_ENABLE = yes
+LINK_TIME_OPTIMIZATION_ENABLE = yes
+BOOTMAGIC_ENABLE = lite
```
Comment (drashna) [inline keyboards/nightly_boards/n87/keymaps/via/rules.mk:3]: Please use the short form of LTO here.
Also, if you enable VIA, it enables bootmagic lite. And you already have it enabled at the keyboard level. 
```suggestion
LTO_ENABLE = yes
```

File: keyboards/nightly_boards/n87/rules.mk [hunk 2]
```diff
@@ -15,18 +15,18 @@ BOOTLOADER = atmel-dfu
...
-BLUETOOTH_ENABLE = no       # Enable Bluetooth
-AUDIO_ENABLE = yes           # Audio output
-FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+NKRO_ENABLE = yes             # USB Nkey Rollover
+BACKLIGHT_ENABLE = no         # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes         # Enable keyboard RGB underglow
+MIDI_ENABLE = no              # MIDI support
+BLUETOOTH_ENABLE = no         # Enable Bluetooth
+AUDIO_ENABLE = yes            # Audio output
+FAUXCLICKY_ENABLE = no        # Use buzzer to emulate clicky switches
```
Comment (fauxpark) [inline keyboards/nightly_boards/n87/rules.mk:31]:
```suggestion
BLUETOOTH_ENABLE = no         # Enable Bluetooth
AUDIO_ENABLE = yes            # Audio output
```

Comment (NightlyBoards) [reply keyboards/nightly_boards/n87/rules.mk]: https://github.com/qmk/qmk_firmware/pull/9885/commits/85db5dc34a97bb83d74a031fb505ce8fcfc2cbba

File: keyboards/nightly_boards/n87/keymaps/via/rules.mk (hunk 1 above)
Comment (NightlyBoards) [reply keyboards/nightly_boards/n87/keymaps/via/rules.mk]: https://github.com/qmk/qmk_firmware/pull/9885/commits/85db5dc34a97bb83d74a031fb505ce8fcfc2cbba

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**